packages/muelu/src/Graph/StructuredAggregation/MueLu_AggregationStructuredAlgorithm_def.hpp

```diff
@@ -249,7 +249,7 @@ namespace MueLu {
     myGraph = CrsGraphFactory::Build(rowMap,
                                      colMap,
                                      nnzOnRow,
-                                     Xpetra::DynamicProfile);
+                                     Xpetra::StaticProfile);
```

Comment:
@lucbv should really comment on this.

Reply:
My feeling is: this should be fine, the graph constructor actually allocates the exact amount of memory it needs since nnzOnRow is specified.
Of course I would still want this example to be tested to double check correctness.

Reply:
Thanks @lucbv :-)